packages/vue/src/components/templates/SfShipping/SfShipping.stories.js

```diff
@@ -0,0 +1,104 @@
+import { SfShipping } from "@storefront-ui/vue"
+import { countries, shippingMethodsData } from "../internalData.js"
```

Comment:
There is also no ShippingMethods in internalData file.
![Screenshot from 2021-09-28 16-04-47](https://user-images.githubusercontent.com/32042425/135103041-37a0a3b3-95ee-4a99-b054-4dc2f456cb48.png)